app/models/global_state.rb

```diff
@@ -58,4 +61,16 @@ def self.valid_new_incident_years
     # for last year iff submission hasn't closed out for the year.
     submission_closed_for_year? ? [Time.current.year] : [Time.current.year - 1, Time.current.year]
   end
+
+  def self.maintenance_mode?
+    instance.maintenance_mode
```

Comment:
Ah, actually I need to do a `read_attribute` (or, synonymously, `[]`) to return nil if it doesn't exist, because the `maintenance_mode` method would be created dynamically.